tests/numpy/numpy_test.py

```diff
@@ -1,6 +0,0 @@
-from base_test import ArkoudaTest
-
-
-class StatsTest(ArkoudaTest):
-    def setUp(self):
-        ArkoudaTest.setUp(self)
```

Comment:
Should we just delete this file, or is my github just not showing it has been removed

Reply:
It's been removed.

Reply:
To clarify, I deleted the file rather than moving to the tests/deprecated folder because there were no tests implemented, so it didn't seem worth saving.

Reply:
Thank you, it just wasnt showing up on my end